test/unit/lib/plugins/aws/lib/updateStack.test.js

```diff
@@ -197,6 +197,23 @@ describe('updateStack', () => {
       });
     });
 
+    it('should include custom stack policy during updates', () => {
+      awsDeploy.serverless.service.provider.stackPolicyDuringUpdate = [
```

Comment:
We're moving away from these old-style tests and instead all new tests suites should be written using `runServerless` utility e.g. like here: https://github.com/serverless/serverless/blob/21aeda4482b4128c0e8df9bf7b35315964c99751/test/unit/lib/plugins/aws/deployFunction.test.js#L908

Reply:
done.  I refactored the test to use runServerless